npm/vue/src/index.ts

```diff
@@ -323,7 +323,21 @@ export function mount<
   >
 >
 
-// implementation
+/**
+ * Mounts a component and returns an object containing the component and VueWrapper
+ * @param componentOptions
+ * @param options
+ * @returns {{vueWrapper: VueWrapper, component: ComponentPublicInstance}}
+ *
+ * @example
+ * import { mount } from '@cypress/vue'
+ * import { Stepper } from './Stepper.vue'
+ * it('mounts', () => {
+ *   cy.mount(Stepper)
+ *   cy.get('[data-cy=increment]').click()
+ *   cy.get('[data-cy=counter]').should('have.text', '1')
+ * })
+ */
 export function mount (componentOptions: any, options: any = {}) {
```

Comment:
`componentOptions` and `options` here should be typed instead of `any`

Reply:
Might need help from Lachlan here. I was under the impression this is the proper way to overload functions in TS

Reply:
This is fine - above this line all the overloads are defined. You then make the implementation typed as `any`. It's working fine:

![image](https://user-images.githubusercontent.com/19196536/199374526-fdccf543-786b-45bc-9c29-8cc47d5bdf0b.png)

Types are pretty messy because of all the overloads, though. :shrug: